GhostfolioSidekick/FileImporter/DeGiro/DeGiroParser.cs

```diff
@@ -146,17 +150,24 @@ protected override CsvConfiguration GetConfig()
 		}
 
 		private decimal GetQuantity(DeGiroRecord record)
-		{
-			var quantity = Regex.Match(record.Omschrijving, $"Koop (?<amount>\\d+) @ (?<price>.*) EUR").Groups[1].Value;
+        {
+            // oop is the same for both buy and sell or Koop and Verkoop in dutch
+            // dont include currency at the end, this can be other things than EUR
+            var quantity = Regex.Match(record.Omschrijving, $"oop (?<amount>\\d+) @ (?<price>[0-9]+,[0-9]+)").Groups[1].Value;
+
 			return decimal.Parse(quantity, GetCultureForParsingNumbers());
 		}
 
-		private decimal GetUnitPrice(DeGiroRecord record)
+		private decimal GetUnitPrice(DeGiroRecord record, bool buy = true)
```

Comment:
not sure why there is an unused parameter 👼

Reply:
you'll have to approve the new changes